public/app/features/alerting/unified/components/amroutes/AmRootRouteForm.tsx

```diff
@@ -104,7 +108,7 @@ export const AmRootRouteForm: FC<AmRootRouteFormProps> = ({
                 <div className={cx(styles.container, styles.timingContainer)}>
                   <InputControl
                     render={({ field, fieldState: { invalid } }) => (
-                      <Input {...field} className={styles.smallInput} invalid={invalid} />
+                      <Input {...field} className={styles.smallInput} invalid={invalid} placeholder={'Time'} />
```

Comment:
Maybe put default as placeholder? See `description` above for what the default is